kyo-prelude/jvm/src/test/scala/kyo2/MonadLawsTest.scala

```diff
@@ -0,0 +1,82 @@
+package kyo2
+
+import zio.Trace
+import zio.prelude.Equal
+import zio.prelude.coherent.CovariantDeriveEqual
+import zio.prelude.coherent.CovariantDeriveEqualIdentityFlatten
+import zio.prelude.laws.*
+import zio.test.*
+import zio.test.laws.*
+
+object MonadLawsTest extends ZIOSpecDefault:
+
+    case class Myo[+T](v: T < (Env[String] & Abort[String] & Sum[Int] & Var[Boolean]))
```

Comment:
I was hoping we wouldn't need this wrapper anymore but the compiler doesn't allow extending interfaces with an `AnyVal` type parameter. The encoding using a context function had the same issue with the compiler not allowing to extend interfaces with a context function type.